counterexamples/direct_sum_is_internal.lean

```diff
@@ -75,7 +75,7 @@ end
 
 /-- But there is no embedding into `ℤ` from the direct sum. -/
 lemma with_sign.not_injective :
-  ¬function.injective (direct_sum.to_module ℕ (units ℤ) ℤ (λ i, (with_sign i).subtype)) :=
+  ¬function.injective (direct_sum.to_module ℕ (ℤˣ) ℤ (λ i, (with_sign i).subtype)) :=
```

Comment:
```suggestion
  ¬function.injective (direct_sum.to_module ℕ ℤˣ ℤ (λ i, (with_sign i).subtype)) :=
```
does this work without the parentheses?

Reply:
If it doesn't, then the precedence might be wrong

Reply:
Ok, I think that this is a minor issue, but the notation would look nicer if parentheses were not needed, at least most of the times.

Reply:
It seems that starting from `1025` the unit symbol starts working without parentheses.

Reply:
I've pushed a commit using the higher precedence.